src/pages/components/accordion/code.mdx

```diff
@@ -1,22 +1,177 @@
 ---
 title: Accordion
-description: The accordion component delivers large amounts of content in a small space through progressive disclosure. The user gets key details about the underlying content and can choose to expand that content.
-tabs: ["Code", "Usage", "Style", "Accessibility"]
+description:
+  The accordion component delivers large amounts of content in a small space
+  through progressive disclosure. The user gets key details about the underlying
+  content and can choose to expand that content.
+tabs: ['Code', 'Usage', 'Style', 'Accessibility']
 ---
 
+import usage from './snippets/usage';
+import skeleton from './snippets/skeleton';
+
+<PageDescription>
+
+Learn how to build an accordion in React.
```

Comment:
Would it be better to explain what an Accordion is briefly here? Thinking about other pages getting a little repetitive: "Learn how to build a[Dropdown,Select,DataTable] in React"

Reply:
Agreed! Don't know what's better though. Happy to remove.

Reply:
Seems like Jan offered a suggestion below 👀 added